### ISSUE TITLE
Fine-tune QuartzNet15x5NR-En to another language [Question]

### ISSUE THREAD

**Issue by Gorodecki** [NVIDIA/NeMo#1855]
Hello!
Thank you for this opensource product!
I want to try fine-tune model **QuartzNet15x5NR-En** to Russian language. This article [Transfer learning from English to Russian](https://developer.nvidia.com/blog/jump-start-training-for-speech-recognition-models-with-nemo/) is good practice for my experiment, but i have some questions.

1. In `config.yaml` in this model `RIR_and_Noise` i find this sample for augmentation:
  > rir_manifest_path: /data/datasets/freesound_20s/rir_noises_tarred/rir_tarred/tarred/tarred_audio_manifest.json
  > rir_tar_filepaths: /data/datasets/freesound_20s/rir_noises_tarred/rir_tarred/tarred/audio_{0..63}.tar
  
   How i download this files and create this manifest?
   I find this script - `download_resample_freesound.sh` - what parameters should be specified for download?

2. How many `hrs` should dataset have? What type of audio data should it contain? (phone calls, audio book...)
3. What `learning rate` to use? What parametrs set to `SpecAug`?

Thank you!

**Comment by Gorodecki** [comment]
@okuchaiev Maybe there is some information on this issue? Thanks!

**Comment by titu1994** [comment]
There are several notebooks for transfer learning char and subword models in Nemo now, please have a look and reopen the issue if they are insufficient.
